src/main/java/com/woowacamp/soolsool/core/liquor/repository/LiquorQueryDslRepository.java

```diff
@@ -55,13 +55,27 @@ public List<LiquorElementResponse> getCachedList(
         final Pageable pageable
     ) {
         log.info("LiquorQueryDslRepository getCachedList");
-        return getList(
-            LiquorSearchCondition.nullObject(),
-            pageable,
-            null
-        );
+        return getList(pageable, null);
```

Comment:
```suggestion

        return getList(null, pageable, null);
```

아래 메서드 추가 없이 이것만 사용해도 될 것 같아요!

Reply:
그렇게 하면 condition.get 하는 부분에서 npe가 터질거 같아요!

Reply:
맞네요... 대신 다른 방법이 생각나 다른 곳에 comment 달았습니다.